### DISCUSSION TITLE
[Feature] Read-only Gallery - extension exclusion

### DISCUSSION THREAD

**Discussion by davidXire** [immich-app/immich#3664] [Feature Request]
### The feature

Hello,

I often keep the RAW files and produce an associated JPG to have the rendering linked to the development and make it easier to view using third-party tools. Would it be possible to add a feature or exclude extensions from the indexing?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by etnoy** [comment]
Coming soon with https://github.com/immich-app/immich/pull/3124
